android/src/test/java/com/segment/analytics/kotlin/android/AndroidAnalyticsKtTest.kt

```diff
@@ -0,0 +1,20 @@
+package com.segment.analytics.kotlin.android
+
+import org.junit.jupiter.api.Assertions.*
+
+import org.junit.jupiter.api.Test
+
+internal class AndroidAnalyticsKtTest {
+    @Test
+    fun `jvm initializer in android platform should failed`() {
+        try {
```

Comment:
i think the new junit has a way to expect exceptions instead of doing this pattern. if u wanna try that

Reply:
let me try that